packages/create-widget.template.react.v2/templates/src/client.ts.hbs

```diff
@@ -0,0 +1,8 @@
+import { $ontologyRid } from "{{osdkPackage}}";
+import { createClient } from "@osdk/client";
+
+export const client = createClient(
+    window.location.origin,
+    $ontologyRid,
+    () => Promise.resolve("widgets-auth"),
+);
```

Comment:
nit: this is kind of hard to keep up without enforcing it as I don't think the template includes a formatter out of the box but we can keep the indentation consistent with the current template files:

https://github.com/palantir/osdk-ts/blob/5553fadd006a3364535b6d0a0add2cd07d1999ef/packages/create-widget.template.react.v2/templates/src/context.ts

```suggestion
export const client = createClient(
  window.location.origin,
  $ontologyRid,
  () => Promise.resolve("widgets-auth"),
);
```